libnmstate/nm/applier.py

```diff
@@ -194,4 +200,5 @@ def _build_connection_profile(iface_desired_state, base_con_profile=None):
         ovs_port_options = iface_desired_state.get('options')
         settings.append(ovs.create_port_setting(ovs_port_options))
 
-    return connection.create_profile(settings)
+    new_profile = connection.create_profile(settings)
```

Comment:
Why is this change needed?

Reply:
it makes debugging easier because it allows to add `import pdb; pdb.set_trace()` after the assignment before the function returns.

Reply:
If you'll need to embed such lines in the code, then you can split it when you actually debug.
Anyway, with this reasoning we will have to double or triple the lines of code in any program, I prefer to be driven by readability and cleanness and not by a rare need of using a debugger. A temporary variable is nice only if it gives the reader more information (and sometime if it helps in avoiding breaking a line into two).

Reply:
I prefer temporary variables in general because they allow to digest the code in smaller pieces. I can also remove this change if you prefer it.

Reply:
It is a nit, I will not fight over it.